### DISCUSSION TITLE
Move Frame by Frame and 48bit Images

### DISCUSSION THREAD

**Discussion by Jay123210599** [mifi/lossless-cut#1896] [Support]
How do I move only 1 frame forward and backward by using the left and right keys on my keyboard? Are there already hotkeys for this, and if so, how do I change them? Also, how do I make 24bit images instead of 48bit images? They're too large for me.

**Comment by mifi** [comment]
use the `,` and `.` keys. can be changed in Settings (keyboard shortcuts)

> Also, how do I make 24bit images instead of 48bit images?

not sure what you mean by that

**Reply by Jay123210599** [reply]
Sorry for the late reply, but what I mean is 24bit (8 x 3 for RGB) and 48 (16 x 3 RGB). I only want the former.

**Reply by mifi** [reply]
you mean when using the "capture screenshot" function? that's not possible to customise currently

**Reply by Jay123210599** [reply]
I mean when extracting frames.

**Reply by mifi** [reply]
I've added your suggestion here: #1908
